docs/userGuide/components/presentation.md

```diff
@@ -20,6 +20,15 @@
 
 The components in this page are the core **presentational** components you may want to use. Panels and tabs can be used to **organise content sections**, while badges and boxes can **highlight small, specific pieces of information**.
 </div>
+<box type = "warning" header = "#### Use of markdown in content" >
```

Comment:
A bit confused by these documentation changes - are these part of your PR?

Reply:
It is not part of this PR;
It is part of another PR that it merged, not sure why it appears here.
Fixed after pressing "update this branch".